apps/argus/src/config/withdraw_fees.rs

```diff
@@ -0,0 +1,28 @@
+use {
+    crate::{api::ChainId, config::ConfigOptions},
+    clap::Args,
+};
+
+#[derive(Args, Clone, Debug)]
+#[command(next_help_heading = "Withdraw Fees Options")]
+#[group(id = "Withdraw Fees")]
+pub struct WithdrawFeesOptions {
+    #[command(flatten)]
+    pub config: ConfigOptions,
+
+    /// Withdraw the fees on this chain, or all chains if not specified.
+    #[arg(long = "chain-id")]
+    pub chain_id: Option<ChainId>,
+
+    /// If provided, run the command using the keeper wallet. By default, the command uses the provider wallet.
+    /// If this option is provided, the keeper wallet must be configured and set as the fee manager for the provider.
+    #[arg(long = "keeper")]
+    #[arg(default_value = "false")]
+    pub keeper: bool,
```

Comment:
nit: the name of this parameter is confusing, you could name it `withdrawer` and make it an enum of either 'provider' or 'keeper'.